Packages/MIES/MIES_DataAcquisition.ipf

```diff
@@ -9,6 +9,27 @@
 /// @file MIES_DataAcquisition.ipf
 /// @brief __DQ__ Routines for Data acquisition
 
+/// @brief Stop DAQ and TP on all locked devices
+Function DQ_StopOngoingDAQAllLocked()
+	string panelTitle
+
+	variable i, numDev
+	string device
+
+	SVAR devices = $GetDevicePanelTitleList()
+	numDev = ItemsInList(devices)
+	for(i = 0; i < numDev; i += 1)
+		device = StringFromList(i, devices)
+
+		try
+			ClearRTError()
```

Comment:
@t-b what's giving rise the RT error (when a new experiment is being initiated)?

Reply:
@timjarsky If you load old MIES experiments the window hook responsible for unlocking and cleaning up does bug out on old panels. This is expected. But pretty annoying with the new uncompile hook.